pkg/loop/internal/common/utils.go

```diff
@@ -0,0 +1,32 @@
+package common
+
+import (
+	"math"
+
+	libocr "github.com/smartcontractkit/libocr/offchainreporting2plus/types"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/loop/internal/pb"
+)
+
+func PbReportTimestamp(ts libocr.ReportTimestamp) *pb.ReportTimestamp {
```

Comment:
How about `package pb` with:
```suggestion
func FromReportTimestamp(ts libocr.ReportTimestamp) *pb.ReportTimestamp {
```
and
```go
func ToReportTimestamp(ts *pb.ReportTimestamp) (r libocr.ReportTimestamp, err error) {
```
Oh actually, we already have an `internal/pb`. Would these fit in there?

Reply:
yes, moved, thanks